client/src/pages/EditOrganisationAccount.js

```diff
@@ -454,6 +539,9 @@ function EditOrganisationAccount({ refetchUser, history }) {
               {renderSelectItems()}
               <Label>{t("profile.org.seeking")}</Label>
               <HelpWrapper>{renderNeedSection()}</HelpWrapper>
+              <WhiteSpace />
+              <Label>{t("profile.org.offering")}</Label>
```

Comment:
You'll need to translate this with the `<Trans>` component like in `CreateOrganisationProfile`

```
                   <Trans i18nKey="profile.org.offering">
                      <b>
                        offer
                      </b>
                    </Trans>
```